spec/presenters/result_set_presenter_spec.rb

```diff
@@ -105,6 +107,7 @@
       presenter.to_hash[:documents].present?.should == true
       presenter.to_hash[:applied_filters].present?.should == true
       presenter.to_hash[:any_filters_applied].present?.should == true
+      presenter.to_hash[:atom_url].present?.should == true
```

Comment:
For the sake of consistency this line is fine, but as an aside, I think you can use `presenter.to_hash[:atom_url].should be_present` here for a slightly nicer phrasing